### DISCUSSION TITLE
如何实现微软拼音里打词选字的功能

### DISCUSSION THREAD

**Discussion by Cirn09** [rime/home#850] [Q&A] [answered]
微软拼音里可以打一个词，然后按<kbd>-</kbd> / <kbd>=</kbd> 输入这个词的前/后一个字
比如我打“比如”，按下<kbd>-</kbd>就是“比”，按下<kbd>=</kbd>就是“如”

这个功能在rime应该可以用lua实现，但具体不会写

**Comment by liang-0131** [comment, marked as answer]
就是以词定字功能可以看[这里](https://github.com/BlindingDark/rime-lua-select-character)
